CSS/components/navigation.css

```diff
@@ -0,0 +1,25 @@
+.nav-bar{
+    width: 100%;
+    display: flex;
+    padding: 0.5rem ;
+    align-items: center;
+    box-shadow: 1px 2px 25px 1px var(--light-gray);
+    background-color: var(--background-color);
+}
+.nav-left{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 1rem;
+}
+.nav-right{
+    display: flex;
+    gap: 1rem;
+    margin-left: auto;
+}
```

Comment:
Hey, can you let me know what this margin left auto is doing, as I think auto works when there is fixed width and I don't see it in your styles. Let me know.

Reply:
The auto keyword will give the left side a share of the remaining space.

Reply:
ya margin auto is setting both icon which is inside the nav-right to the right corner as vikram suggestion to use margin-left auto in navbar to set the icon to right